docs/entity_registry_index.md

```diff
@@ -11,26 +11,30 @@ A user is also able to override the name of an entity in the entity registry. Wh
 
 ## Unique ID
 
-It is important that it is not possible for the user to change the unique ID, because they system would lose all its settings related to it.
+It is important that it is not possible for the user to change the unique ID, because the system would lose all its settings related to it.
```

Comment:
```suggestion
It is important that it is not possible for the user to change the unique ID, because the system would lose all its settings related to the unique ID.
```